modules/core/src/lib/init.js

```diff
@@ -0,0 +1,12 @@
+import {global} from '@loaders.gl/loader-utils';
+// @ts-ignore TS2304: Cannot find name '__VERSION__'.
+const version = typeof __VERSION__ !== 'undefined' ? __VERSION__ : '';
+
+// @ts-ignore
+global.loaders = Object.assign(global.loaders || {}, {
+  VERSION: version,
```

Comment:
Is it necessary to have both?

Reply:
Yes... I suspect this may have been copied from deck.gl which I believe has both. 

How about we use `VERSION` only here, since we have no legacy and we use UPPER_CASE for constants.

Reply:
Ok. Removed lower_case `version`.